### PR TITLE
Update README, CHANGELOG, and reword warnings on ServiceWorker mode.

### LINKED ISSUE #421: Reword the warnings about ServiceWorker mode
So far, it was unstable so we stated serious warnings to discourage its use, except for testing.
Now it's a bit different : the ServiceWorker mode is more stable.
But there are still a few issues. At least : 
- recent wikimedia ZIM files, when viewed on small screens, have the sections collapsed, with no possibility to open them
- when switching between jQuery and ServiceWorker modes, I still manage to break the UI. Workaround is to reload the page or app. I've created #423 for that

### PR DESCRIPTION
**Pull request by mossroy**
Fixes #421

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,11 +4,18 @@ Please note that this application has changed its name over time.
...
-## Kiwix-JS v2.4.0
+## Kiwix-JS v2.5.0
 Released on TODO
 
 TODO
 
+## Kiwix-JS v2.4.0
+Released on 2018-09-22
+
+Performance improvement on decompression of content (now twice faster)
```
Comment (Jaifroid) [inline CHANGELOG.md]: > (now twice faster) 

"(now twice as fast)" is better English

File: www/index.html [hunk 2]
```diff
@@ -217,17 +218,16 @@ <h3>Expert settings</h3>
...
                                                 <input type="radio" name="contentInjectionMode" value="jquery" id="jQueryModeRadio" checked>
-                                                <strong>JQuery</strong> (slow and memory hungry, but safer)
+                                                <strong>JQuery</strong> (stable and safer)
                                             </label>
                                         </div>
                                         <div class="radio">
                                             <label>
                                                 <input type="radio" name="contentInjectionMode" value="serviceworker" id="serviceworkerModeRadio">
-                                                <strong>ServiceWorker</strong> (faster but unstable, and not supported by all platforms)
+                                                <strong>ServiceWorker</strong> (still a bit experimental, and not supported by all platforms)
```
Comment (Jaifroid) [inline www/index.html]: > still a bit experimental, and not supported by all platforms 

English is fine, but maybe could be more concise: "experimental, not supported by all platforms".

File: www/index.html [hunk 3]
```diff
@@ -11,10 +11,11 @@
         Home page : http://www.kiwix.org
 
         Main authors of this application :
-        Mossroy - mossroy@mossroy.fr
+        Mossroy - https://github.com/mossroy - mossroy@mossroy.fr
         Peter-x - https://github.com/peter-x
+        Jaifroid - https://github.com/Jaifroid
```
Comment (Jaifroid) [inline www/index.html]: Thank you! Very kind of you to add me!

File: CHANGELOG.md (hunk 1 above)
Comment (mossroy) [reply CHANGELOG.md]: OK thanks, updated

File: www/index.html (hunk 2 above)
Comment (mossroy) [reply www/index.html]: To me, the SW mode is now usable in most cases (when available). It's not really "experimental" any more, even if we know a few edge cases that cause problems.
So I'd keep the "still a bit", unless you have a better way to say that?

File: www/index.html (hunk 3 above)
Comment (mossroy) [reply www/index.html]: That's normal : I should have done that earlier.
Thanks again for all the work you do on this project.

File: www/index.html (hunk 2 above)
Comment (Jaifroid) [reply www/index.html]: @mossroy: OK, so maybe "Somewhat experimental …."? I think "Still a bit" implies that the user knows how it used to be, which is not necessarily the case. But I'm fine with "Still a bit" if you prefer it.

Comment (mossroy) [reply www/index.html]: OK, updated with "somewhat experimental" and pushed

**Comment by mossroy** [comment]
I've merged this (but forgot to squash before : oops)

**Pull request merged**